scheduler/src/cook/components.clj

```diff
@@ -230,6 +231,16 @@
       (log/info "Found framework id in zookeeper:" framework-id)
       framework-id)))
 
+(defn auth-filter
+  "Skip authentication on some hard-coded endpoints."
+  [h auth-middleware]
+  (let [auth-fn (auth-middleware h)]
+    (fn filtered-auth [req]
+      (if (and (= (:uri req) "/info")
```

Comment:
Nitpick: prefer to see the constant first in the equals comparison.

Reply:
Done